cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,88 @@
-describe('', () => {
+describe('The task Article Flow', () => {
   before(() => {
-
+    cy.visit('/');
   });
-
-  it('', () => {
-
+  it('Main Page', () => {
+    // User is on the main Pgae
+    cy.contains('Conduit')
+      .should('exist');
+    cy.contains('a', 'Global Feed')
+      .should('exist');
+    cy.contains('.sidebar p', 'Popular Tags')
+      .should('exist');
+    cy.get('.nav-link')
+      .contains('Sign in')
+      .should('be.visible');
   });
-
-  it('', () => {
-
+  // User is log in
+  it('Successfull login', function () {
+    cy.visit('/user/login');
+    cy.login('cy.test@email.com', '123456789');
```

Comment:
Use functions provided by Mate. The original one takes 3 arguments. 
User should be randomly generated each time.